.changeset/heavy-terms-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-actions': patch
+---
+
+`Add missing token on job list call to GitHub API
```

Comment:
```suggestion
Add missing token on job list call to GitHub API
```

Reply:
Fixed in b8e73f3b97a8141c71f848fc56ce4b69df5e4217